packages/creator/test/creator.test.ts

```diff
@@ -283,4 +253,218 @@ describe("Module `creator`", () => {
       });
     });
   }
+
+  describe("createSolutionFromItemIds", () => {
+    it("createSolutionFromItemIds fails to create solution item", done => {
+      const itemIds: string[] = [];
+      const authentication: common.UserSession = MOCK_USER_SESSION;
+      const url =
+        "https://myorg.maps.arcgis.com/sharing/rest/content/users/casey/addItem";
+
+      fetchMock.post(url, { success: false });
+      spyOn(common, "createId").and.callFake(() => "xfakeidx");
```

Comment:
Interesting...I had to read on this a bit to follow. Thats a cool pattern that will come in handy